### PR TITLE
added topics for robot-name

### PR DESCRIPTION
**Pull request by PatrM**
added topics for #920. 
I feel like something is missing though. What do you think @FridaTveit?

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -434,7 +434,11 @@
       "core": true,
       "difficulty": 5,
       "slug": "robot-name",
-      "topics": null,
+      "topics": [
+        "regular_expressions",
+        "strings",
+        "randomness"
```
Comment (FridaTveit) [inline config.json]: Maybe `text_formatting` and `pattern_matching` could be useful? I think you've included the most important ones though, so I don't mind if you add those two or not :)

Comment (PatrM) [reply config.json]: `text_formatting` is a good idea, I gonna add that later on. 
Isn't `pattern_matching` already covered by regex though?

Comment (FridaTveit) [reply config.json]: You could try and look for something matching a pattern without using a regex. And some people might be looking for exercises to do with pattern matching without thinking to look for exercises to do with regular expressions. But you're right, they are very similary so I'm happy for you to just stick with `regular_expressions` if you think that's best :)

**Comment by FridaTveit** [comment]
Will merge as soon as the build finishes :)

**Pull request merged**